streaming/streaming.go

```diff
@@ -115,6 +136,15 @@ type StreamingCfg struct {
 	DataShardId   string                   `json:"dataShardId"`
 }
 
+// Returns the retry error codes and backoff policy to the GCP client retry logic.
+func dataStreamGaxRetrier() gax.Retryer {
+	return gax.OnCodes(DEFAULT_DATASTREAM_RETRY_CODES, gax.Backoff{
+		Initial:    DEFAULT_DATASTREAM_CLIENT_BACKOFF_BASE_DELAY,
+		Max:        DEFAULT_DATASTREAM_CLIENT_BACKOFF_MAX_DELAY,
+		Multiplier: DEFAULT_DATASTREAM_CLIENT_BACKOFF_MULTIPLIER,
+	})
```

Comment:
What are the total number of retries that will be performed?

Reply:
Total Number of retries will be around 15 
$$log2(900) / log2(1.6)$$